pkg/reconciler/awscodecommitsource/adapter.go

```diff
@@ -50,8 +47,10 @@ type adapterConfig struct {
 // adapterDeploymentBuilder returns an AdapterDeploymentBuilderFunc for the
 // given source object and adapter config.
 func adapterDeploymentBuilder(src *v1alpha1.AWSCodeCommitSource, cfg *adapterConfig) common.AdapterDeploymentBuilderFunc {
+	adapterName := common.AdapterName(src)
```

Comment:
This change feels like a stretch to this PR, but because the `FindAdapter` method uses this new `AdapterName()` method under the hood, I figured it would make sense to also use it the adapter builder funcs right away, in a separate commit.

Reply:
So this is to ensure that the adapterName conforms to knative's expectations going forward?

Reply:
Knative doesn't expect any particular format, I just needed a way to derive the "app" name from the Kubernetes object so I don't have to pass it as an argument to every function call.
Also we don't have to hard code some arbitrary name anymore, it's always gonna be the object type lower cased (which is what we used until now anyway).